vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -52,27 +47,19 @@ std::string ConvertTypeToString(const ledger::RewardsType type) {
 }
 
 void GenerateSuggestionMock(
-    ledger::UnblindedTokenPtr token,
+    ledger::UnblindedToken token,
```

Comment:
`const ledger::UnblindedToken&` ?

Reply:
nice catch, it was pointer before and I replace it last minute